README.md

```diff
@@ -36,6 +36,24 @@ You can follow the instructions there to make sure your tool is working as expec
 yarn run docs:gen
 ```
 
+## Limitations
+
+- `remark-validate-links` is only supported with `repository: false`, details
```

Comment:
Codacy found an issue: [[write-good] "validate" is wordy or unneeded](https://app.codacy.com/gh/codacy/codacy-remark-lint/pullRequest?prid=6212959)